pkg/chunk/schema.go

```diff
@@ -58,6 +58,9 @@ type IndexQuery struct {
 
 	// Filters for querying
 	ValueEqual []byte
+
+	// If the result of this lookup can be cached or not.
+	Cacheable bool
```

Comment:
Yes, but the only answer to that is `0` and `infinite` because how long we can cache the active entries is determined by how long we keep chunks which the schema has no info about.

Reply:
Right, `false` doesn't mean don't cache, it means cache based on something else.  Therefore this is misleading.  Maybe this should be called `Mutable`, indicating it will change, and therefore shouldn't be cached for too long?